contracts/src/internal/HyperdriveStorage.sol

```diff
@@ -139,9 +131,17 @@ abstract contract HyperdriveStorage is ReentrancyGuard {
 
     /// Constructor ///
 
+    // TODO: It would be good to update `PoolConfig` when we make other breaking
+    //       changes.
```

Comment:
I know I'm not telling you anything you don't know, but any changes to poolConfig are going to cause issues with the frontend. They will have to find an efficient way to load different SDKs for different pools. This could bloat the bundle size unless they come up with something efficient.

Reply:
Yep, 100%. I get it. This is just to remind us to update the pool config with this information when we feel that it is appropriate.